### PR TITLE
Sidebar

### PR DESCRIPTION
**Pull request by ro-sa**

### CODE REVIEW

File: src/components/Sidebar.vue [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+    height="100vh"
+    floating
+    disable-route-watcher
+    fixed
+    temporary
+  >
+    <v-list-item class="px-2">
+      <v-list-item-avatar>
+        <v-img src="/assets/logo.svg" contain alt="logo"></v-img>
+        <!-- <v-icon>mdi-account</v-icon> -->
```
Comment (Exilliar) [inline src/components/Sidebar.vue:17]: comment can be deleted if not needed

File: src/components/Sidebar.vue [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+  }
+
+  changeRoute(newPath: string) {
+    if (this.$route.path !== newPath) {
+      this.$router.push(newPath);
+    }
+    return;
+  }
+
+  data() {
```
Comment (Exilliar) [inline src/components/Sidebar.vue:96]: this `data` function isn't needed I think. The `props` field can just be done using the decorators (like on line 72), then the items can just be declared like a normal vairable like in other components.
Happy to jump on a call if you want me to explain more

File: src/App.vue [hunk 3]
```diff
@@ -62,6 +70,17 @@ export default class App extends Vue {
     return this.$store.state.darkMode;
   }
 
+  data() {
```
Comment (Exilliar) [inline src/App.vue:73]: see other comment about `data` function

**Review by Exilliar — COMMENTED** [review]
looking good. Couple of comments.

Also, I'm getting the thing where there's different shades of black on the login page (if that makes sense). I remember you talking about this issue, can't remember if it's this pr that's meant to fix it or another one

**Comment by ro-sa** [comment]
> looking good. Couple of comments.
> 
> Also, I'm getting the thing where there's different shades of black on the login page (if that makes sense). I remember you talking about this issue, can't remember if it's this pr that's meant to fix it or another one

background color of login page v-cards fixed. ready to merge now.

**Review by Exilliar — APPROVED** [review]
looks good. We def need to fix the stuff going on with the landing page. But that can be done in another issue

**Pull request merged**